File: classwork/classwork01/CtoFconverter.c
#include <stdio.h>
#include <stdlib.h>

int main( int argc, char * argv[] ) {

   double degreesF;
   char   input[25];
   printf( "Enter a temperature in degrees Fahrenheit: " );
   degreesF = atof( gets( input ) );
   printf( "%10.3f degrees F is %10.3f degrees C. ", degreesF, (((degreesF - 32.0) * 5.0) / 9.0) );
   exit( 0 );

}

File: test/test1/CtoFconverter.java
import java.util.Scanner;
class CtoFconverter {
   public static void main( String args[] ) {
      System.out.println( "Enter a temperature in degrees Fahrenheit: ");
      Scanner myInput = new Scanner( System.in );
      double degreesF = Double.parseDouble(myInput.nextLine());
      System.out.println( degreesF + " degrees F is " + (((degreesF - 32.0) * 5.0) / 9.0) + " degrees C." );
   }
}